### PR TITLE
zh translation: faq/_index.md

### PR DESCRIPTION
**Pull request by zc2638**
Signed-off-by: zc <ce.zheng@daocloud.io>

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/faq/_index.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: FAQ
+title: 常见问题
```
Comment (kebe7jun) [inline content/zh/faq/_index.md:2]: FAQ 不用翻译，这是社区通用的叫法。

**Review by kebe7jun — CHANGES_REQUESTED** [review]

File: content/zh/faq/_index.md (hunk 1 above)
Comment (zc2638) [reply content/zh/faq/_index.md:2]: 参考其他文档，均已将FAQ翻译

Comment (kebe7jun) [reply content/zh/faq/_index.md:2]: 这是放在 header 上的，和文档里面的描述不属于一个层级的，我觉得应该不需要翻译。

**Pull request closed without merging**